### PR TITLE
feat: royalty free images

### LINKED ISSUE #16: replace images
all images should be replaced with something with proper licensing

### PR DESCRIPTION
**Pull request by mikejgray**
chore: sync skill.json to locale folder

Closes #16

## Summary by CodeRabbit

- **New Features**
	- Introduced a new job in the workflow to automate synchronization of the `skill.json` file.
	- Updated the `README.md` with new image references and added a "License" section.
	- Enhanced `skill.json` with updated image paths and new credits for AI-generated images.

- **Bug Fixes**
	- Corrected image paths in `skill.json` and `README.md` to ensure proper display.

### CODE REVIEW

File: .github/workflows/update_skill_json.yml [hunk 1]
```diff
@@ -8,3 +7,22 @@ jobs:
     permissions:
       contents: write
     uses: neongeckocom/.github/.github/workflows/skill_update_json_spec.yml@master
```
Comment (JarbasAl) [inline .github/workflows/update_skill_json.yml:9]: not the right spec anymore

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 Makes your voice assistant laugh like a maniac
 
-![A ghostly face, laughing evilly.](./sshot.png)
+![A ghostly face, laughing evilly.](./ui/images/2.jpg)
```
Comment (JarbasAl) [inline README.md:10]: images should be moved to "gui/all", this ensures in docker setups they are available across containers as the "gui" folder gets copied to a shared volume

Comment (JarbasAl) [reply README.md:10]: similarly, in the code we should no longer pass absolute paths to allow the cached path to resolve instead

` self.gui.show_image(join(dirname(__file__), "ui", "images", str(pic) + ".jpg"))` -> ` self.gui.show_image(str(pic) + ".jpg"))`

Comment (mikejgray) [reply README.md:10]: Do we have this documented somewhere? Wasn't aware this was an option

Comment (mikejgray) [reply README.md:10]: Looks like this is in #38

File: .github/workflows/update_skill_json.yml (hunk 1 above)
Comment (mikejgray) [reply .github/workflows/update_skill_json.yml:9]: The important parts are there, so leaving for now

File: .github/workflows/update_skill_json.yml [hunk 3]
```diff
@@ -8,3 +7,22 @@ jobs:
...
+    runs-on: ubuntu-latest
+    needs: update-skill-json
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Sync skill.json
+        run: |
+          cp skill.json locale/skill.json
```
Comment (JarbasAl) [inline .github/workflows/update_skill_json.yml:21]: should be locale/en-us

**Pull request merged**